### DISCUSSION TITLE
Vivaldi support?

### DISCUSSION THREAD

**Discussion by Gregoras** [extesy/hoverzoom#1433] [Ideas]
Vivaldi is a lesser-known Chrome derivate from former Opera browser devs, focused mostly on privacy.
Vivaldi has pretty good compatibility with Chrome extensions, but Hoover Zoom is causing problems, because sometimes (not always) previews look like that: 
![image](https://github.com/user-attachments/assets/7ba92b71-2aae-45a6-b83b-6a0e8715856f)

**Comment by extesy** [comment]
If you or someone else could help with specific pull requests, that would be great. We do not have enough time to keep up with Chrome and Firefox, so unfortunately Vivaldi (and Brave, Opera, etc) is far down the list of priorities.
